src/main/java/com/aws/greengrass/clientdevices/auth/metrics/ClientDeviceAuthMetrics.java

```diff
@@ -64,8 +67,8 @@ public ClientDeviceAuthMetrics(Clock clock) {
      * Emit metrics using Metric Factory.
      */
     public void emitMetrics() {
-        // TODO need to call this function on a timer
         List<Metric> retrievedMetrics = collectMetrics();
+        logger.atDebug().kv("Emitting metrics: ", retrievedMetrics);
```

Comment:
you need to log this.

`logger.atDebug().kv("metrics", retrivedMetrics).log("Emitting CDA metrics")`